e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -172,7 +154,7 @@ const URL_WITH_FILLED_PARAMS = URL_WITH_PARAMS.replace(
   .replace(`{{${CREATED_AT_COLUMN_ID}}}`, POINT_CREATED_AT)
   .replace(`{{${DASHBOARD_FILTER_TEXT.slug}}}`, FILTER_VALUE);
 
-describe("scenarios > dashboard > dashboard cards > click behavior", () => {
+describeEE("scenarios > dashboard > dashboard cards > click behavior", () => {
```

Comment:
IDK how it worked before. `setTokenFeatures` is called in `beforeEach` and that function will throw without EE.